pkg/helmexec/exec_test.go

```diff
@@ -362,10 +362,7 @@ func Test_DecryptSecret(t *testing.T) {
 
 	_, err := helm.DecryptSecret(HelmContext{}, "secretName")
 	if err != nil {
-		if _, ok := err.(*os.PathError); ok {
```

Comment:
why change?

Reply:
@philomory

Reply:
I changed that because the new implementation doesn't return a PathError.

Reply:
To be frank, I don't fully understand why the test was the way it was initially. I looked in the commit history but there was no explanation included with the PR that added it.

Reply:
@philomory @raxod502-plaid Thanks very much.